src/fixes/extra_blank_line.rs

```diff
@@ -0,0 +1,179 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct ExtraBlankLineFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for ExtraBlankLineFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "ExtraBlankLine",
+        }
+    }
+}
+
+impl Fix for ExtraBlankLineFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let warning_count = warnings.len();
+
+        // check and remove all blank lines.
+        lines.dedup_by(|a, b| a.is_empty() && b.is_empty());
+
+        // mark as fixed
+        for warning in warnings {
+            warning.mark_as_fixed();
+        }
+
+        Some(warning_count)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    #[test]
+    fn fix_multiple_lines_test() {
```

Comment:
```suggestion
    fn no_blank_lines_test() {
```
It will be more obvious.